cli/cmd/cluster_kubeconfig.go

```diff
@@ -64,6 +65,17 @@ func (r *runners) kubeconfigCluster(_ *cobra.Command, args []string) error {
 		return errors.New("must provide cluster id or name")
 	}
 
+	cluster, err := r.kotsAPI.GetCluster(clusterID)
+	if errors.Cause(err) == platformclient.ErrForbidden {
```

Comment:
I don't think that endpoint ever returns StatusForbidden

Reply:
I will clean up in the next PR